src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String OPERATION_TYPE_BUY = "buy";
+    private static final String OPERATION_TYPE_SUPPLY = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> records = new ArrayList<>();
```

Comment:
The variable 'records' is unnecessary since its contents are not used after being filled. You can process each line inside the while loop in the 'readFromFile' method.